manim/utils/tex_templates.py

```diff
@@ -0,0 +1,1022 @@
+# -*- coding: utf-8 -*-
+"""A library of LaTeX templates."""
+__all__ = [
+    "TexTemplateLibrary",
+    "TexFontTemplates",
+]
+
+from .tex import *
+
+
+# This file makes TexTemplateLibrary and TexFontTemplates available for use in manim Tex and MathTex objects.
+
+
+def _new_ams_template():
+    # Returns a simple Tex Template with only basic AMS packages
+    preamble = r"""
+\usepackage[english]{babel}
+\usepackage{amsmath}
+\usepackage{amssymb}
+"""
+    return TexTemplate(preamble=preamble)
+
+
+# TexTemplateLibrary
+#
+class TexTemplateLibrary(object):
+    """
+    A collection of basic TeX template objects
+
+    Usage
+    -----
+    As a value for the keyword argument tex_template of Tex() and MathTex() mobjects
+        Tex("My TeX code", tex_template=TexTemplateLibrary.ctex)
+
+    """
+
+    default = TexTemplate()
+    """ An instance of the default TeX template in manim """
```

Comment:
the guidelines recommend bundling the attrs in the class docstring, but this particular case is debatable

Reply:
"bundling the attrs in the class docstring"
I have to admit I have no idea what these words mean.

Reply:
i.e., having one Attributes section in the class's docstring, instead of distributing the docstrings of each attribute to under them

Reply:
ah yes. I see. Makes sense. Noted for future.
But with these particular classes (which just serve as a collection of attributes) perhaps that is suboptimal.

Reply:
yeah, I guess we can open an exception here then

Reply:
let's remove those spaces
```suggestion
    """An instance of the default TeX template in manim"""
```